src/components/templates/MainContent.vue

```diff
@@ -0,0 +1,17 @@
+<script setup>
+import ContentHeader from '@/components/molecules/ContentHeader.vue'
+import AddTodo from '@/components/molecules/AddTodo.vue'
+import TodoList from '@/components/organisms/TodoList.vue'
+import TodoActions from '@/components/molecules/TodoActions.vue'
+import AppHeader from '@/components/atoms/AppHeader.vue'
+</script>
+
+<template>
+  <AppHeader />
```

Comment:
Why not to create page? A page is the final, fully realized design step. It looks like the Page should be in App.vue, not template. It should be in separate folder - pages. Template should be more like layout, and page is a final view, that users see